### PR TITLE
Added SeriManga presence

### LINKED ISSUE #3518: Seri Manga
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [x] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?**  Golgomosh#3465

**What's the name of the service?** https://serimanga.com/ | Seri Manga

**What should the Presence display?** website name,name of the manga being read and part number (like this - https://imgur.com/ssMPKje )

**If possible, please provide a logo for the service (512x512 minimum)**  ( https://imgur.com/h4SVq8k )
Screenshot_2
seri-manga_1481923

### PR DESCRIPTION
**Pull request by MareGraphics**
Fully working Serimanga.com presence.
resolve #3518
Screenshots:
image
image
image
image

### CODE REVIEW

File: websites/S/SeriManga/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "url": "serimanga.com",
+    "description": {
+      "en": "Türkçe Manga ve Çizgi Romanları ücretsiz ve kaliteli bir şekilde okuyabileceğiniz site."
+    },
+    "settings": [
+      {
+        "id": "ID",
+        "multiLanguage": true
+      }
+    ]
```
Comment (ririxidev) [inline websites/S/SeriManga/dist/metadata.json:26]: you don't even use it so why you need it?

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/S/SeriManga/dist/metadata.json (hunk 1 above)
Comment (MareGraphics) [reply websites/S/SeriManga/dist/metadata.json:26]: It was there for testing, guess i forgot about it

File: websites/S/SeriManga/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "version": "1.0.0",
+    "color": "#2d2f33",
+    "tags": [
+      "manga",
+      "manhwa"
+    ],
+    "url": "serimanga.com",
+    "description": {
+      "en": "Türkçe Manga ve Çizgi Romanları ücretsiz ve kaliteli bir şekilde okuyabileceğiniz site."
+    }
```
Comment (Bas950) [inline websites/S/SeriManga/dist/metadata.json:20]: that doesn't look English to me...

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/S/SeriManga/dist/metadata.json (hunk 2 above)
Comment (MareGraphics) [reply websites/S/SeriManga/dist/metadata.json:20]: Ohh, didn't notice :/

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**